aiida_workgraph/engine/workgraph.py

```diff
@@ -146,7 +148,20 @@ def load_instance_state(
         self.set_logger(self.node.logger)
 
         if self._awaitables:
-            # this is a new runner, so we need to re-register the callbacks
+            # this is a new Process instance
+            # For the "ascyncio.tasks.Task" awaitable, because there are only in-memory,
+            # we need to re-run the task
```

Comment:
```suggestion
            # For the "ascyncio.tasks.Task" awaitable, because there are only in-memory,
            # we need to re-run the task and by that creating a new process instance
```
Is this what you mean? Maybe you can connect the two sentences in some logical way.

Reply:
thanks! I revised the text.